examples/graph_sage_unsup.py

```diff
@@ -108,11 +92,11 @@ def test():
     val_acc = clf.score(out[data.val_mask], data.y[data.val_mask])
     test_acc = clf.score(out[data.test_mask], data.y[data.test_mask])
 
-    return val_acc, test_acc
+    return val_acc, test_acc, out
```

Comment:
`val_acc` and `test_acc` are reduced when I ran it. Also it was little bit slower than the original examples (but that could be because of my machine).

Reply:
Thanks.